unit1_array.js

```diff
@@ -1,29 +1,28 @@
 //////////////////PROBLEM 1////////////////////
 /* Create a copy of the faveColors array called 'colorCopy' using the slice method. */
 
-const faveColors = ['red', 'green', 'black']
+const faveColors = ["red", "green", "black"];
 
 //CODE HERE
-const colorCopy = faveColors.slice()
+const colorCopy = faveColors.slice();
 
 //////////////////PROBLEM 2////////////////////
 /* Add a fourth color to the end of the 'colorCopy' array using the push method. */
 
 //CODE HERE
-faveColors.push('blue')
+faveColors.push("blue");
 
 /////////////////PROBLEM 3////////////////////
 /*
   Using an array method, create a new array called 'middleNums' from the 
   numbers array that will capture only the middle numbers (2, 3, 4).
 */
 
-const numbers = [1, 2, 3, 4, 5]
+const numbers = [1, 2, 3, 4, 5];
 
 //CODE HERE
-let middleNums = number.slice(1,4)
-
-
+//need to be very careful with your syntax
+let middleNums = numbers.slice(1, 4);
```

Comment:
I changed number to numbers variable names need to be the same watch your spelling